packages/vuetify/src/labs/VDateInput/VDateInput.tsx

```diff
@@ -0,0 +1,128 @@
+// Styles
+import './VDateInput.sass'
+
+// Components
+import { VDialog } from '@/components/VDialog'
+import { VMenu } from '@/components/VMenu'
+import { makeVTextFieldProps, VTextField } from '@/components/VTextField/VTextField'
+import { VDateCard, VDatePicker } from '@/labs/VDatePicker'
+
+// Composables
+import { createDateInput, dateEmits, makeDateProps } from './composables'
+import { useDisplay } from '@/composables'
+
+// Utilities
+import { ref, watch } from 'vue'
+import { genericComponent, propsFactory, useRender } from '@/util'
+
+// Types
+import type { VTextFieldSlots } from '@/components/VTextField/VTextField'
+
+export type VDateInputSlots = VTextFieldSlots
+
+export const makeVDateInputProps = propsFactory({
+  mobile: Boolean,
+
+  ...makeDateProps(),
+  ...makeVTextFieldProps({
+    appendInnerIcon: '$calendar',
+    dirty: true,
+    placeholder: 'mm/dd/yyyy',
+  }),
+}, 'VDateInput')
+
+export const VDateInput = genericComponent<VDateInputSlots>()({
+  name: 'VDateInput',
+
+  props: makeVDateInputProps(),
+
+  emits: {
+    ...dateEmits,
+  },
+
+  setup (props, { slots }) {
+    const { mobile } = useDisplay()
+    const { adapter, model, inputMode, viewMode, displayDate, parseKeyboardDate } = createDateInput(props, false)
+
+    const dialog = ref(false)
+    const inputModel = ref(model.value.length ? adapter.format(model.value[0], 'keyboardDate') : '')
+
+    function onBlur () {
+      const { isEqual } = adapter
+      const date = parseKeyboardDate(inputModel.value)
```

Comment:
This assumes US format, so with en-AU the month and day get swapped when you click on a date. There doesn't seem to be a way to specify the input format for `Date.parse()`, it only really works with US or ISO.

Reply:
Temporal doesn't support parsing either: https://tc39.es/proposal-temporal/docs/strings.html#localized-parsing

Reply:
We are adding a default for now and will further address as part of our regular patch cycle.